client/src/components/ChatBox.tsx

```diff
@@ -50,8 +50,10 @@ class ChatBox extends React.Component<Props, State> {
   };
 
   handleNewConversation = (convoPromise: Promise<Response>) => {
-    convoPromise.then(res => res.json())
+    convoPromise
+      .then(res => res.json())
       .then(res => {
+        console.log(res);
```

Comment:
Now that we're not getting this error, let's remove the console.log

Reply:
Okay. done 👍